update-completed-sprint-on-issue-closed/workflow.yml

```
@@ -0,0 +1,90 @@
+name: Update CompletedSprint on Issue Closed
+
+on:
+  issues:
+    types: [closed]
+  pull_request:
+    types: [closed]
```

❗ For this workflow to be callable it needs to have this trigger. You'll create a workflow in each repo that needs to call this repo using the issues and pull_request triggers.

```suggestion
on:
  workflow_call:
```

---

Like this? https://github.com/stellar/soroban-tools/pull/558

---

Yup!